### PR TITLE
Adding prediction.py

### PR DESCRIPTION
**Pull request by valli180**
Please review the code for in the prediction.py in notebooks

### CODE REVIEW

**Review by christopheralex — CHANGES_REQUESTED** [review]
1. Could change the PR graph to use this function which displays AP score. 
`from sklearn.metrics import PrecisionRecallDisplay`
`PrecisionRecallDisplay.from_estimator(pipe_lr, X_valid, y_valid);`
2. No need for reading train in script , only read test and change only y label to 0 and 1 . Followed by predicting using the loaded model. 
3. Read data from preprocessed instead of raw. 

You can use the same PR to push to your branch for re review. 

Rest all look fine.

**Review by christopheralex — CHANGES_REQUESTED** [review]
Everything looks fine. But delete the classification report, confusion matrix and other graphs which are not in the results directory

**Review by christopheralex — APPROVED** [review]
